tests/test_state.py

```diff
@@ -57,6 +57,15 @@ def test_default_provider(state_instance):
 def test_multiple_platforms(state_instance):
     assert not state_instance.multiple_platforms
 
+def test_customconf(state_instance):
```

Comment:
I think YAPF will fail due to missing 2 lines between methods.  Might wanna re-run the formatter.